api/src/main/kotlin/com/google/devtools/ksp/processing/Resolver.kt

```diff
@@ -212,4 +221,35 @@ interface Resolver {
      */
     @KspExperimental
     fun getDeclarationsFromPackage(packageName: String): Sequence<KSDeclaration>
+
+    /**
+     * Returns the corresponding Kotlin class with the given Java class.
+     *
+     * E.g.
+     * java.lang.String -> kotlin.String
+     * java.lang.Integer -> kotlin.Int
+     * java.util.List -> kotlin.List
+     * java.util.Map.Entry -> kotlin.Map.Entry
+     * java.lang.Void -> null
+     *
+     * @param javaName a Java class name
+     * @return corresponding Kotlin class name or null
+     */
+    @KspExperimental
+    fun mapJavaToKotlin(javaName: KSName): KSName?
```

Comment:
maybe change name to `mapJavaNameToKotlin`? Same for the other one.

Reply:
Done.